dashboard/v2/.bowerrc

```diff
@@ -1,5 +1,5 @@
 {
-  "directory": "dist/lib",
+  "directory": "public/lib",
```

Comment:
Why it should be in target? Its required runtime and not only compile time like java classes. While creation of war dist folder gets added to it. Dist folder is create from public folder with minified files.
If we move it to target folder then every mvn clean it will get deleted and which will need bower to download it again. Which defeats the purpose of whole offline build support.